addons/controllers/suite_test.go

```diff
@@ -79,6 +79,7 @@ var (
 	cancel        context.CancelFunc
 	certPath      string
 	keyPath       string
+	tmpDir        string
 	// clientset     *kubernetes.Clientset
```

Comment:
question: this might not be from this PR, but could we remove the commented line in above ?

Reply:
done